to.holepunch.Keet.yml

```diff
@@ -0,0 +1,52 @@
+id: to.holepunch.Keet
+runtime: org.gnome.Platform
```

Comment:
why do you need the gnome platform here since it's Electron based.

Reply:
the installer for the Pear platform uses native GUI for its splash screen

Reply:
define "native GUI"

Reply:
it calls into GTK

Reply:
so basically you don't need. gtk3 is part of the freedesktop runtime.